website/docs/docs/collaborate/project-recommendations.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Project recommendations"
+sidebar_label: "Project recommendations"
+description: "dbt Explorer provides recommendations that you can take to improve the quality of your dbt project."
+---
+
+:::tip Public preview 
+
+The project recommendations feature is now available in dbt Explorer! Check it out!
+
+:::
+ 
+dbt Explorer provides recommendations about your project from the `dbt_project_evaluator` [package](https://hub.getdbt.com/dbt-labs/dbt_project_evaluator/latest/) using metadata from the Discovery API.
+
+Explorer also offers a global view, showing all the recommendations across the project for easy sorting and summarizing. This overview page includes two top-level metrics measuring the test and documentation coverage of the models in your project. 
+
+- **Model test coverage** &mdash; The percent of models in your project (models not from a package or imported via dbt Mesh) with at least one dbt test configured on them.
+- **Model documentation coverage** &mdash; The percent of models in your project (models not from a package or imported via dbt Mesh) with a description.
+
+These recommendations provide insight into how you can build a more well documented, well tested, and well built project, leading to less confusion and more trust.
```

Comment:
Ooh definitely move this sentence up so google searches find it easily!

Reply:
great idea!